trellis/trellis.go

```diff
@@ -38,10 +38,6 @@ func (t *Trellis) Detect(path string) (projectPath string, ok bool) {
 func (t *Trellis) CreateConfigDir() error {
 	_, err := os.Stat(t.ConfigPath)
```

Comment:
Maybe this should just try to always to `Mkdir` and then check if the error is `IsExist`? If it's not, return the error. Something like

```go
if err := os.Mkdir(t.ConfigPath, 0755); err != nil {
  if !os.IsExist(err) {
    return err
  }
}

return nil
```

Reply:
Good idea, one less syscall, super optimized!